GetMessages/__tests__/handler.test.ts

```diff
@@ -142,6 +143,21 @@ const messageStatusModelMock = ({
   findLastVersionByModelId: mockFindLastMessageStatusVersion
 } as unknown) as MessageStatusModel;
 
+const setWithExpirationTaskMock = jest
```

Comment:
Maybe should check in some tests if setWithExpirationTaskMock is called.

Reply:
It's already present in https://github.com/pagopa/io-functions-app-messages/blob/14c6a6d02de12303be02975b5d2e10a2c6d514b4/utils/__tests__/messages.test.ts#L243

Reply:
sorry, I missed it!